gufe/setup/chemical_system_generator.py

```diff
@@ -0,0 +1,25 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+import abc
+from enum import Enum
+
+from typing import Iterable
+from gufe import ChemicalSystem
+
+# Todo: connect to protocols - use this for labels?
+
+class RFEComponentLabels(str, Enum):
```

Comment:
This needs more discusison, I don't remember this being in the plans for updating things, definitely using `labels` for components seems like something we don't want to do.

Reply:
this is part of OpenFE Currently. however I wondered why, because it would make more sense here in my oppinion.
(I think at some time point we thought this is better then plain strings ;) )

Reply:
I don't think it should be migrated here, this seems like an implementation rather than an abstract class.

Reply:
remove this again.